packages/phrases/src/locales/en/errors.ts

```diff
@@ -180,6 +180,9 @@ const errors = {
     scope_exists: 'The scope id {{scopeId}} has already been added to this role',
     user_exists: 'The user id {{userId}} is already been added to this role',
```

Comment:
nit: might be a little bit unrelated to this PR, but these should be revised. `is already been` is grammarly incorrect